v22.2/trigram-indexes.md

```diff
@@ -0,0 +1,360 @@
+---
+title: Trigram Indexes
+summary: Trigram indexes on STRING columns improve the performance of fuzzy string matching and pattern matching in large datasets.
+toc: true
+keywords: gin, gin index, gin indexes, inverted index, inverted indexes, accelerated index, accelerated indexes, trigram, trigrams
+docs_area: develop
+---
+
+{% include_cached new-in.html version="v22.2" %} A _trigram index_ is a type of [inverted index](inverted-indexes.html) created on a [`STRING`](string.html) column. Trigram indexes are used to efficiently search for strings in large tables without providing an exact search term.
+
+This page describes how to create and use trigram indexes on CockroachDB.
+
+{{site.data.alerts.callout_info}}
+Some PostgreSQL syntax and features are currently unsupported. For details, see [Unsupported features](#unsupported-features).
+{{site.data.alerts.end}}
+
+## How do trigram indexes work?
+
+Trigram indexes make [substring and similarity matches](https://www.postgresql.org/docs/current/pgtrgm.html) efficient by indexing the unique trigrams of a string. A trigram is a group of three consecutive characters (including spaces) in a string.
+
+To display all of the possible trigrams for a string, use the `show_trgm()` [built-in function](functions-and-operators.html#trigrams-functions):
```

Comment:
nit: it's not the possible trigrams, it's all of the trigrams. Maybe say "to display the trigrams within a string"?